backend/routes.py

```diff
@@ -8,5 +8,5 @@
 
 @blueprint.route('/recycling-data')
 def get_recycling_data():
-    data = [row.to_dict() for row in db_client.get_all_classroom_data()]
+    data = [row.to_dict() for row in db_client.get_all_recycling_data()]
```

Comment:
We're returning data as an array atm. Maybe it should be a dict object key'd by id